src/EventStore.ClientAPI.Embedded/EventStoreEmbeddedNodeConnection.cs

```diff
@@ -375,10 +375,11 @@ public Task<AllEventsSlice> ReadAllEventsBackwardAsync(Position position, int ma
                                                                          Action<EventStoreCatchUpSubscription> liveProcessingStarted = null,
                                                                          Action<EventStoreCatchUpSubscription, SubscriptionDropReason, Exception> subscriptionDropped = null,
                                                                          UserCredentials userCredentials = null,
-                                                                         int readBatchSize = 500)
+                                                                         int readBatchSize = 500,
+                                                                         string connectionLogSubscriptionName = "")
```

Comment:
Can you change `connectionLogSubscriptionName` here to just `subscriptionName` to avoid confusion